rclpy/rclpy/impl/rmw_implementation_tools.py

```diff
@@ -30,8 +30,11 @@ def reload_rmw_implementations():
     global __rmw_implementations
     __rmw_implementations = sorted(ament_index_python.get_resources('rmw_implementation').keys())
 
-    # Remove implementations that are being filtered for in the rclp CMakeLists so they cannot be selected
-    __rmw_implementations = [rmw_impl for rmw_impl in __rmw_implementations if rmw_impl not in ['rmw_connext_dynamic_cpp', 'rmw_fastrtps_cpp']]
+    # Remove implementations that are being filtered for in the rclpy CMakeLists so
```

Comment:
Nitpick: I would wrap after `CMakeLists` then the next line makes more sense on its own.